src/DI/ServiceDefinition.php

```diff
@@ -12,6 +12,10 @@
 
 /**
  * Definition used by ContainerBuilder.
+ *
+ * @property $class
+ * @property $factory
+ * @property-read $entity
```

Comment:
@dg Is this correct? Also there are other private properties with getters and setters. Should I add all of them? (These seem to be important.)